meadControllerNodeWithWebApp/meadControllerNodeWithWebApp.ino

```diff
@@ -66,19 +67,19 @@ void loop()
   if (writeCounter == 60)
   {
     writeCounter = 0;
-    
+
     if (WiFiMulti.run() == WL_CONNECTED)
-    {  
```

Comment:
My editor did this automatically, but you should configure your editor to catch rogue whitespaces like this. I don't do this, but some people look at code with whitespace characters on